internal/ui/logger.go

```diff
@@ -1,12 +1,22 @@
 package ui
 
 import (
+	"github.com/dream11/odin/pkg/constant"
 	log "github.com/sirupsen/logrus"
+	"os"
 )
 
 func init() {
+
 	log.SetFormatter(&log.TextFormatter{
-		ForceColors:   true,
-		DisableColors: false,
+		ForceColors:     true,
+		DisableColors:   false,
+		TimestampFormat: "2006-01-02 15:04:05", // Custom format
+		FullTimestamp:   true,
 	})
+	if os.Getenv(constant.LogLevelKey) == "yes" {
```

Comment:
Instead of taking `yes` take actual log level itself

Reply:
Then keeping `ODIN_LOG_LEVEL` as key

Reply:
Yes